source/__tests__/integration/export-profile.test.js

```diff
@@ -4,6 +4,10 @@ describe('Sinopia Profile Editor exports an edited profile', () => {
     await page.goto('http://localhost:8000/#/profile/create/true')
   })
 
+  afterAll(async () =>{
+      await page.waitFor(1000)
+  })
```

Comment:
again - this is an unconditional wait that will slow down the tests, even when they don't need to be slowed down.  Better to wait for a selector with this timeout or something like that.